unumpy/_multimethods.py

```diff
@@ -1421,3 +1564,153 @@ def _vander_default(x, N=None, increasing=False):
 @all_of_type(ndarray)
 def vander(x, N=None, increasing=False):
     return (x,)
+
+
+@create_numpy(_first2argreplacer)
+@all_of_type(ndarray)
+def split(ary, indices_or_sections, axis=0):
+    return (ary, indices_or_sections)
+
+
+@create_numpy(_first2argreplacer)
+@all_of_type(ndarray)
+def array_split(ary, indices_or_sections, axis=0):
+    return (ary, indices_or_sections)
+
+
+@create_numpy(
+    _first2argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=2),
+)
+@all_of_type(ndarray)
+def dsplit(ary, indices_or_sections):
+    return (ary, indices_or_sections)
+
+
+@create_numpy(
+    _first2argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=1),
+)
+@all_of_type(ndarray)
+def hsplit(ary, indices_or_sections):
+    return (ary, indices_or_sections)
+
+
+@create_numpy(
+    _first2argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=0),
+)
+@all_of_type(ndarray)
+def vsplit(ary, indices_or_sections):
+    return (ary, indices_or_sections)
+
+
+def _tile_default(A, reps):
+    if isinstance(reps, numbers.Integral):
+        reps = (reps,)
+
+    prep = len(reps) - ndim(A)
+    if prep > 0:
+        for _ in range(prep):
+            A = expand_dims(A, 0)
+    elif prep < 0:
+        reps = tuple(itertools.repeat(1, abs(prep))) + reps
+
+    for axis, dim in enumerate(reps):
+        dim = operator.index(dim)
+        if dim == 0:
+            return asarray([], dtype=A.dtype)
+
+        tile = A
+        for _ in range(dim - 1):
+            A = concatenate((A, tile), axis=axis)
+
+    return A
+
+
+@create_numpy(_first2argreplacer, default=_tile_default)
+@all_of_type(ndarray)
+def tile(A, reps):
+    return (A, reps)
+
+
+@create_numpy(_first2argreplacer)
+@all_of_type(ndarray)
+def repeat(a, repeats, axis=None):
+    return (a, repeats)
+
+
+@create_numpy(_first2argreplacer)
+@all_of_type(ndarray)
+def delete(arr, obj, axis=None):
+    return (arr, obj)
+
+
+def _insert_argreplacer(args, kwargs, dispatchables):
+    def replacer(arr, obj, values, axis=None):
+        return dispatchables, {"axis": None}
+
+    return replacer(*args, **kwargs)
+
+
+@create_numpy(_insert_argreplacer)
+@all_of_type(ndarray)
+def insert(arr, obj, values, axis=None):
+    return (arr, obj, values)
```

Comment:
`obj` isn't dispatchable.

Reply:
The replacer will need modifying too.

Reply:
You are right, I probably misread the docs.

Reply:
I think the same is true for `delete`.